Web/Admin.UI/src/app/components/core/vd-icon/vd-icon.component.html

```
@@ -0,0 +1,16 @@
+<i class="vd-icon" [ngClass]="class">
+  <ng-container *ngIf="isSvg; else imgTemplate">
+    <ng-container *ngIf="svgContent !== null; else loadingTemplate">
```

HTMLHint (1.5.0)

[error] The attribute name of [ *ngIf ] must be in lowercase.

(attr-lowercase)